index.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My_Portfolio</title>
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.2/css/all.css"
+        integrity="sha384-oS3vJWv+0UjzBfQzYUhtDYW+Pj2yciDJxpsK1OYPAYjqT085Qq/1cq5FLXAZQ7Ay" crossorigin="anonymous" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,400;0,500;0,700;1,400&display=swap"
+        rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css"
+        integrity="sha512-NhSC1YmyruXifcj/KFRWoC561YpHpc5Jtzgvbuzx5VozKpWvQ+4nXhPdFgmx8xqexRcpAglTj9sIBWINXa8x5w=="
+        crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <header>
+        <div class="container-header">
+            <h1 class="logo-title">Marlyn Mayienga</h1>
+            <nav class="nav-bar">
+                <ul>
+                    <li><a href="#"></a>Portfolio</a></li>
+                    <li><a href="#"></a>About</a></li>
+                    <li><a href="#"></a>Contact</li>
+                    <li><a href="#"><i class="far fa-envelope"></i></a></li>
+                </ul>
+            </nav>
+            <div class="hamburger-btn">
+                <i class="fas fa-bars"></i>
+            </div>            
+        </div>
+    </header>
+
+    <main class="content">
+        <h1 class="name">Marlyn Mayienga</h1>
+        <img class="first-img" src="../My_Portfolio/circle 2 header.svg" alt="circle 2 header">
```

Comment:
- [ ] This image is not displaying in the project so kindly fix it.
- [ ] your `hr` tag is too big kindly fix it.
-
![img](https://user-images.githubusercontent.com/81727988/164344302-0fe4912b-0757-4e3c-941d-71461b7f2bda.png)